docs/sdk/src/guides/enable_elastic_scaling.rs

```diff
@@ -0,0 +1,154 @@
+//! Parachains can adjust their usage of core resources on the fly to increase TPS and decrease latency.
+//! 
+//! ### When do you need Elastic Scaling?
+//! 
+//! Depending on their use case, applications might have an increased need for the following:
+//! - compute (CPU weight)
+//! - bandwidth (proof size)
+//! - lower latency (block time)
+//! 
+//! ### High throughput (TPS) and lower latency
+//! 
+//! If the main bottleneck is the CPU, then your parachain needs to maximize the compute usage of each core while also achieving 
+//! a lower latency.
+//! 3 cores provide the best balance between CPU, bandwidth and latency: up to 6s of execution, 3MB/s of DA bandwidth and fast 
+//! block time of just 2 seconds.
+//! 
+//! ### High bandwidth 
+//! 
+//! Useful for applications that are bottlenecked by bandwidth.
+//! By using 6 cores, applications can make use of up to 6s of compute, 6MB/s of bandwidth every 6s while also achieving 
+//! 1 second block times. 
+//! 
+//! ### Ultra low latency
+//! When latency is the primary requirement, Elastic scaling is currently the only solution. The caveat is
+//! the efficiency of core time usage decreases as more cores are used. 
+//! 
+//! For example, using 12 cores enables fast transaction confirmations with 500ms blocks and up to 12 MB/s of DA bandwidth.
+//!
+//! ## Dependencies
+//! 
+//! Prerequisites: Polkadot-SDK `2509` or newer.
+//! 
+//! To ensure the security and reliability of your chain when using this feature you need the following:
+//! - An omni-node based collator. This has already become the default choice for collators.
+//! - RFC 103. This is mandatory protection against PoV replay attacks.
+//! - Enabling the relay parent offset feature. This is required to ensure the parachain block times 
+//!   and transaction in-block confidence is not negatively affected by relay chain forks.
+//! - Block production configuration adjustments.
+//! 
+//! ### Upgrade to Polkadot Omni node
+//! 
+//! Your collators need to run `polkadot-parachain` or `polkadot-omni-node` with the `--authoring slot-based` CLI argument.
+//! 
+//! Further information about omni-node and how to upgrade is available:
+//! - [high level docs](https://docs.polkadot.com/develop/toolkit/parachains/polkadot-omni-node/)
+//! - [`crate::reference_docs::omni_node`]
+//! 
+//! ### Enable RFC103
+//!
+//! RFC103 is enabled automatically on the collators if it is enabled on the relay chain. There are no code changes
+//! required on the client to support it. 
+//! 
+//! RFC103 relies on the ability of parachain blocks to commit to a specific core index on the relay chain.
+//! This commitment is implemented via `UMP` signals, which rely on the upward message passing channel that
+//! is used by parachains to send messages to the relay chain.
+//! 
+//! The only required change for the runtime is enabling the `experimental-ump-signals` feature of the `parachain-system`
```

Comment:
Why is this still experimental? When will we no longer make it experimental?

Reply:
Because it is not enabled on Polkadot yet. I plan to  remove this feature completely as soon as RFC103 is enabled on Polkadot.